cloudbuild-publish.yaml

```diff
@@ -0,0 +1,33 @@
+steps:
+  - name: 'gcr.io/cloud-builders/gcloud'
+    script: |
+      #!/usr/bin/env bash
+      set -e
+      REPO_TOKEN="$(gcloud auth print-access-token)"
+      cat > ~/.npmrc <<EOL
+      //registry.npmjs.org/:_authToken=$NPM_TOKEN"
+      registry=https://us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/
+      //us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/:always-auth=true
+      //us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/:_authToken=${REPO_TOKEN}
+      EOL
+    secretEnv: ['NPM_TOKEN']
+  - name: gcr.io/cloud-builders/bazel:5.4.0
+    script: |
+      #!/usr/bin/env bash
+      set -e
+      bazel run @nodejs//:yarn config set registry https://us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/
+      ./scripts/publish
```

Comment:
Presumably we should only ever run the publish script when using the airlock registry from now on. So we should just update `./scripts/publish` to do all of this

Reply:
👍